quantum/rgblight.h

```diff
@@ -244,73 +235,4 @@ void rgblight_effect_christmas(void);
 void rgblight_effect_rgbtest(void);
 void rgblight_effect_alternating(void);
 
-#endif // #ifndef RGBLIGHT_H_DUMMY_DEFINE
 #endif // RGBLIGHT_H
```

Comment:
This needs to be here. 
```suggestion
#endif // #ifndef RGBLIGHT_H_DUMMY_DEFINE	
#endif // RGBLIGHT_H
```